### PR TITLE
chore: openlayers 6 api support

### PR DESCRIPTION
**Pull request by AnguIar**

### CODE REVIEW

File: docs/getting-started/raster/raster_step-by-step.md [hunk 1]
```diff
@@ -306,33 +306,22 @@ Replace all `<>` place holders with the real values that we got from all previou
 ...
 ...
 ...
+    const parser = new WMTSCapabilities();
+    const capabilitiesResponse = await fetch('<RASTER-RASTER-SERVING-SERVICE_URL>/wmts/1.0.0/WMTSCapabilities.xml');      // from Step_4
```
Comment (asafMasa) [inline docs/getting-started/raster/raster_step-by-step.md:310]: Why don't you use the build-in capabilities URL from links?

The following should be changed to 
<mc:links scheme="WMTS" name="[desired_layer_identifier]" description=""
url=[capabilities-url]>
</mc:links>
and then reference the link

File: docs/getting-started/raster/raster_step-by-step.md [hunk 2]
```diff
@@ -306,33 +306,22 @@ Replace all `<>` place holders with the real values that we got from all previou
...
+    const capabilitiesText = await capabilitiesResponse.text();
+    const parserResult = parser.read(capabilitiesText);
+    const layerOptions = optionsFromCapabilities(parserResult, {
+      layer: '<LAYER_NAME>'                                                                                               // from Step_3
```
Comment (asafMasa) [inline docs/getting-started/raster/raster_step-by-step.md:314]: <LAYER_NAME> -> [desired_layer_identifier] from links

**Review by asafMasa — CHANGES_REQUESTED** [review]

**Review by syncush — CHANGES_REQUESTED** [review]
fix conflicts and rebase

File: docs/getting-started/raster/raster_step-by-step.md [hunk 3]
```diff
@@ -314,33 +311,22 @@ Replace all `<>` place holders with the real values that we got from all previou
...
 ...
 ...
 ```
-- extent - value should be the BBOX (extent) that you got from [Step 2](#step-2).
-- url - should be replaced by the URL that you got from [Step 3](#step-3) or [Step 4](#step-4).
-- layer - should be replaced with layer Product ID.
-- matrixSetID -from Response from [Step 4](#step-4).
-- style - should be replaced with the value that you got from [Step 4](#step-4).
-- format - should be replaced with the value that you got from [Step 4](#step-4).
+- When fetching data, make sure to provide correct **WMTS Capabilities URL**
```
Comment (asafMasa) [inline docs/getting-started/raster/raster_step-by-step.md:328]: add extent of BBOX from CSW to layer

Comment (asafMasa) [inline docs/getting-started/raster/raster_step-by-step.md:328]: check if notes are needed (Capabilities, desired_layer_identifier)

**Review by asafMasa — CHANGES_REQUESTED** [review]

**Review by asafMasa — APPROVED** [review]

**Review by syncush — APPROVED** [review]

**Pull request merged**